Description: dmidecode.8 Fix groff error
Author: Soren Stoutner <soren@debian.org>
Forwarded: https://savannah.nongnu.org/patch/index.php?10463
Last-Update: 2024-07-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/man/dmidecode.8
===================================================================
--- trunk.orig/man/dmidecode.8
+++ trunk/man/dmidecode.8
@@ -1,3 +1,5 @@
+'\" t
+.\" ** The above line should force tbl to be a preprocessor **
 .TH DMIDECODE 8 "February 2023" "dmidecode"
 .\"
 .SH NAME
